packages/ui/src/2_molecules/Dialog/dialog.module.css

```diff
@@ -0,0 +1,32 @@
+@import "../../../styles/index.css";
+
```

Comment:
Rename the file to `Dialog.module.css`, make sure all of the components have proper file names.